cmd/create/cluster/cmd.go

```diff
@@ -1871,7 +1880,24 @@ func run(cmd *cobra.Command, _ []string) {
 	}
 
 	etcdEncryption := args.etcdEncryption
-	if interactive.Enabled() && !fips {
+	etcdEncryptionKmsARN := args.etcdEncryptionKmsARN
+
+	if etcdEncryptionKmsARN != "" {
+		// validate hosted cp
+		if !isHostedCP {
```

Comment:
NP
We can check that earlier after line `677` where `isHostedCP` is set, to fail the user earlier.
Maybe something like:
```
if isHostedCP && args.etcdEncryptionKmsARN != "" {
		r.Reporter.Errorf("etcd encryption kms arn is only allowed for hosted cp")
		os.Exit(1)
	}
```

Reply:
Fixed